harbor/README-v2.13.1.md

```diff
@@ -0,0 +1,18 @@
+# Harbor v2.13.1 Configuration
+
+Below is table that highlights the required fields for the Harbor data values file. 
+
+| Property	                                                        | Value	                             | Description                                                                                                                                                                                                                                                                                                                                   |                                                                                         
+|------------------------------------------------------------------|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|                                                                                        
+| hostname	                                                        | FQDN	                              | The FQDN that you have designated to access the Harbor UI and for referencing the registry in client applications. The domain should be configured in an external DNS server such that it resolves to the Envoy Service IP created by Contour or the External IP of the LoadBalancer Service, depending on the settings.                      |
```

Comment:
depending on the settings. -> depending on the `enableNginxLoadBalancer` and `enableContourHttpProxy` settings.